Doc/library/asyncio-api-index.rst

```diff
@@ -19,7 +19,7 @@ await on multiple things with timeouts.
     :class: full-width-table
 
     * - :func:`run`
-      - Create event loop, run a coroutine, close the loop.
+      - Create event loop, run a coroutine function, close the loop.
```

Comment:
coroutine function -> coroutine object

Reply:
I'm sorry I make a mistake here, I will try to learn more about asyncio before moving forward.

Reply:
NP, it's OK to make mistakes!